docs/algorithms/GenericLineConflation.asciidoc

```diff
@@ -276,7 +276,9 @@ are merged.
 
 [[GenericConflationQuality]]
 .Conflation Quality
-image::algorithms/GenericLineTestGraph.png[Random Forest vs. Generic Line Road Conflation Performance,scalewidth="50%"]
+image::images/GenericLineTestGraph.png[Random Forest vs. Generic Line Road Conflation Performance,scalewidth="50%"]
+
+--------
```

Comment:
This `#TODO:` section should actually be commented out.  It is a previous author's attempt to create the above `GenericLineTestGraph.png` graphic using `gnuplot`.  This can be commented out with a `////` commend block...